HubFramework.xcodeproj/project.pbxproj

```diff
@@ -2128,13 +2800,172 @@
 /* End PBXTargetDependency section */
 
 /* Begin XCBuildConfiguration section */
+		344C5F101E09CF9A00597B61 /* Debug */ = {
+			isa = XCBuildConfiguration;
+			buildSettings = {
+				ALWAYS_SEARCH_USER_PATHS = NO;
```

Comment:
Finally had the chance to look into this. So... I basically just duplicated the setup of the current HubFramework-iOS dynamic target and made my tvOS modifications. All of these overrides are currently in production. 

I will make this modification to both dynamic targets then.